assets/js/components/game.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable import/extensions */
+import display from './display.js';
+import board from './board.js';
+import Player from './player.js';
+
+const game = (() => {
+  const initialTarget = document.querySelector('.card');
+  let turn = 0;
+  let playerXName;
+  let playerOName;
+  let players;
+  let result;
+  const winCombination = [[0, 1, 2], [3, 4, 5], [6, 7, 8],
+    [0, 3, 6], [1, 4, 7], [2, 5, 8],
+    [0, 4, 8], [2, 4, 6]];
+
+  const isEqual = (arr, target) => target.every((v) => arr.includes(v));
+  const winner = () => {
+    // let firstPlayerMove = board.returnBoard; closed in here
+    for (let k = 0; k < players.length; k += 1) {
+      for (let m = 0; m < winCombination.length; m += 1) {
+        console.log(isEqual(players[k].playersMove, winCombination[m]));
+        if (isEqual(players[k].playersMove, winCombination[m])) {
+          console.log('I am from winner');
```

Comment:
Remove all `console.log` in your code base.

Reply:
Remove it from all files